full_utility_rates/California/Petaluma  City Of - 2158/07-01-2017.owrs

```diff
@@ -0,0 +1,129 @@
+author_info:
+  author: 
+  email: 
+  phone: 
+metadata:
+  effective_date: 07/01/2017
+  utility_name: Petaluma  City Of
+  bill_frequency: Monthly
+  bill_unit: ccf
+  prop_218_link: http://cityofpetaluma.net/wrcd/waterrates.html#water
+rate_structure:
+  RESIDENTIAL_SINGLE:
+    service_charge:
+      depends_on:
+        - meter_size
+      values:
+        5/8": 9.57
+        3/4": 9.57
+        1": 9.57
+        1|1/2": 26.46
+        2": 40.98
+        3": 74.83
+        4": 123.3
+        6": 244.06
+    commodity_charge: Tiered
+    tier_starts_commodity:
+      - 0
+      - 4
+      - 8
+      - 16
+    tier_prices_commodity:
+      - 3.52
+      - 3.95
+      - 4.5
+      - 5.35
+    fixed_drought_surcharge: 0
+    variable_drought_surcharge: 0
+    fixed_wastewater_charge: 0
+    variable_wastewater_charge: 0
+    bill: service_charge+commodity_charge
+  RESIDENTIAL_MULTI:
+    service_charge: 5.74*number_dwelling_unit

```

Comment:
Please check if this works.

Reply:
@christophertull going to punt this one over to you. No rush

Reply:
Note SweetWaterSprings has the same issue: https://github.com/California-Data-Collaborative/Open-Water-Rate-Specification/pull/376/files